letsencrypt/plugins/manual.py

```diff
@@ -80,12 +84,7 @@ def prepare(self):  # pylint: disable=missing-docstring,no-self-use
         pass  # pragma: no cover
 
     def more_info(self):  # pylint: disable=missing-docstring,no-self-use
-        return """\
-This plugin requires user's manual intervention in setting up a HTTP
-server for solving SimpleHTTP challenges and thus does not need to be
-run as a privilidged process. Alternatively shows instructions on how
-to use Python's built-in HTTP server and, in case of HTTPS, openssl
-binary for temporary key/certificate generation.""".replace("\n", "")
+        return self.__doc__.replace("\n    ", " ")
```

Comment:
If you take my advice about not using the docstrings here, I think we can do this more cleanly. I'd use the fact that Python [automatically concatenates string literals](https://docs.python.org/2/reference/lexical_analysis.html#string-literal-concatenation). This way, you can cleanly break everything on its own line but have total control over whitespace. How you choose to format it is up to you, but  you could do something like:

```
        return ("This plugin requires user's manual intervention in setting "
                "up a HTTP server for solving SimpleHTTP challenges and thus "
                "does not need to be run as a privilidged process. "
                "Alternatively shows instructions on how to use Python's "
                "built-in HTTP server.")
```